ladybug/src/main/java/org/frankframework/ibistesttool/MessageEncoder.java

```diff
@@ -25,16 +25,28 @@
 import org.apache.commons.io.input.BoundedInputStream;
 import org.apache.commons.io.input.BoundedReader;
 import org.frankframework.stream.Message;
+import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import lombok.SneakyThrows;
 import nl.nn.testtool.Checkpoint;
 import nl.nn.testtool.MessageEncoderImpl;
 
-public class MessageEncoder extends MessageEncoderImpl {
+public class MessageEncoder extends MessageEncoderImpl implements InitializingBean {
 
 	private @Autowired int maxMessageLength;
 
+	// Both Ladybug and the present class in the Frank!Framework
+	// truncate messages as configured through maxMessageLength.
+	// In this class we keep one character more such that
+	// Ladybug can detect when a message is being truncated.
+	private int truncateMessageToLength = -1;
+
```

Comment:
Instead of this extra var and method you could do "maxMessageLengt + MAX_MESSAGE_LENGTH_ADJUSTMENT" where you now used truncateMessageToLength (not sure what would be the best solution)